### DISCUSSION TITLE
Where to get the traditional theme

### DISCUSSION THREAD

**Discussion by hanlins** [plantuml/plantuml#928] [Q&A] [answered]
Hello, I use plantuml in my designs on a daily basis, and I recently found that we can configure the default theme that seems to be updated. I'm trying to keep the style of the old diagrams in my designs consistent with new ones, so I was trying to find the original theme from the [themes](https://github.com/plantuml/plantuml/tree/master/themes) directory. I iterated the themes, but none of them looks the same as the original one (see below for reference):
![test](https://user-images.githubusercontent.com/19394971/154169135-62d22d02-20b0-4c49-9d96-05ae8749795c.png)
I'm wondering is there any place where I can find the original theme just like I showed above? If now, can we create one? Cheers!

**Comment by The-Lum** [comment]
Hello @hanlins,

## 1. A first workaround

You can use the old default skin, here:
- https://github.com/plantuml/plantuml/blob/adc23450c82e822d6fe8836defa01277f7901f71/skin/plantuml.skin

Here is the result with:
>[![](http://www.plantuml.com/plantuml/svg/jLXVRzis47-cFeM0lUwF7DUjinL5azHf7hPXM0RicPREDXUAz4Ywdblak-yeIl5P9zkTWIcrOHx9-zsVtnsfllT1kh2fTVRMXvs6TvcpDkHViZo_joRyAcl87wNnZ-3K0eKFkpMudvLvkhFQkdokDz0kxGIoV6ebZ-epvADNtIYgpzTQAXFrMAS-erJg6wsMfWOJSWsBW5E_susfxgmpuF82nn-KN5eZDH6X0V37IfLF1hpFHzzticJ3DpVJeXY_Hjaj6lFHHNtjpFtiVdOtmvd7bQpiLfbbNk3kvoohRBbfhFWoiE_FbGhHpnN82bopPDW5fvQhw0CDGF4oResODWzAEP2n9k9GRrhHCtw22XfEe9Vu1Gwd-_6d6PXRNJMYtsHLnG1Ccj4lqYsLIGEEJYrbPYsiHTYpTlrlOT6m15CXFbb1g3X0Wm_DCnWeohxHv21GhscWfYruATT1MND1LeZcWceU_USK87qzGe0DM9re2KstGbjKYKlm0GqjnMa1lC82eDpYP4aGIuHUfnNCb6rJ8D4E-7i3feHOj4xMpNAwaQUdNyrNZJaqLgDZHdMw7sXPicJFckS8EiKPdy3HQW5nzqd_EuYqVPBsuhzII-_t4K49vak6Wo150Mfd7BoQj4JR9qzqkHAyiuhdBbrP1Juukqk03XRWOc9mI7s-odXK1jtJa0rdwWJbAIx94SCxaJD-ij1yyMnn4wCLS_iF8Xzl16VK9cAlfPF17d61uvvqwoi0Az1orylMY9-bPpE0PWTLghLCXS_EpibP3R9CRb-I_tdqNhhTXTjBd75MNxWRVwXmAJJsiGrSkBUIGSwbJzlFUQWrbF48yHUcnszLhQL9OCDdn7c-7j1islYWP6rDrJiingcWf7DssoFdE4uO6u39XovYj_UtRzwyyBCzGcCyb2irfD-rCbKjrxHpSfamjahDa1oeUBiGz4Z58M-iISO3ZZrKj0uEBq5jgSLkvwIjz_0fV4mYlzWE9_kmEwVoKv-GnnI99uGj4ftec1SJ7h8-EWzQYWxDcHWwM4LtI2xxN8pRzzJg8OvOx7kJrDLRwU8YWa8Ce4mIp3eMlYn8_uHBUuY9S1fyGDFvT0B_oC0POBmzIHC2oGvnSfY1RIK9WdZT6D1_UOcBwUaaUZOEN19uGGOdZKSTO48coDk0s1CacHLK3JfTX7YIkQA3zxAx0l-ImQMs7hex-_2rhPXDPzDkdhueN1VNq-iqWvhQp0d1oukKu5NwIsB2-cGDRjmd-FZzBYubT60OJFkuQnGR3-NNdGSxMUkM9j_-vQsXLR_lkemqlRld-ATnUsrCtV5ejw5HIrrB7DvqXh54L2t8wx1WLvYq29-LMguqVaAY34jqMIuMHOhT_lAADCC0ydevrRDBQtnmKfbmStOnOfXjfII5C12srYKo7NH1qYz6BmNdFe7ZIt4glUvm9SpnggeE-vrWpUxj3-b_PRfB63AcXApvphzxbz_Q-O_v0sXji_TWgasj_mK0)
_[Click to see the source on PlantUML Online server]_](http://www.plantuml.com/plantuml/umla/jLXVRzis47-cFeM0lUwF7DUjinL5azHf7hPXM0RicPREDXUAz4Ywdblak-yeIl5P9zkTWIcrOHx9-zsVtnsfllT1kh2fTVRMXvs6TvcpDkHViZo_joRyAcl87wNnZ-3K0eKFkpMudvLvkhFQkdokDz0kxGIoV6ebZ-epvADNtIYgpzTQAXFrMAS-erJg6wsMfWOJSWsBW5E_susfxgmpuF82nn-KN5eZDH6X0V37IfLF1hpFHzzticJ3DpVJeXY_Hjaj6lFHHNtjpFtiVdOtmvd7bQpiLfbbNk3kvoohRBbfhFWoiE_FbGhHpnN82bopPDW5fvQhw0CDGF4oResODWzAEP2n9k9GRrhHCtw22XfEe9Vu1Gwd-_6d6PXRNJMYtsHLnG1Ccj4lqYsLIGEEJYrbPYsiHTYpTlrlOT6m15CXFbb1g3X0Wm_DCnWeohxHv21GhscWfYruATT1MND1LeZcWceU_USK87qzGe0DM9re2KstGbjKYKlm0GqjnMa1lC82eDpYP4aGIuHUfnNCb6rJ8D4E-7i3feHOj4xMpNAwaQUdNyrNZJaqLgDZHdMw7sXPicJFckS8EiKPdy3HQW5nzqd_EuYqVPBsuhzII-_t4K49vak6Wo150Mfd7BoQj4JR9qzqkHAyiuhdBbrP1Juukqk03XRWOc9mI7s-odXK1jtJa0rdwWJbAIx94SCxaJD-ij1yyMnn4wCLS_iF8Xzl16VK9cAlfPF17d61uvvqwoi0Az1orylMY9-bPpE0PWTLghLCXS_EpibP3R9CRb-I_tdqNhhTXTjBd75MNxWRVwXmAJJsiGrSkBUIGSwbJzlFUQWrbF48yHUcnszLhQL9OCDdn7c-7j1islYWP6rDrJiingcWf7DssoFdE4uO6u39XovYj_UtRzwyyBCzGcCyb2irfD-rCbKjrxHpSfamjahDa1oeUBiGz4Z58M-iISO3ZZrKj0uEBq5jgSLkvwIjz_0fV4mYlzWE9_kmEwVoKv-GnnI99uGj4ftec1SJ7h8-EWzQYWxDcHWwM4LtI2xxN8pRzzJg8OvOx7kJrDLRwU8YWa8Ce4mIp3eMlYn8_uHBUuY9S1fyGDFvT0B_oC0POBmzIHC2oGvnSfY1RIK9WdZT6D1_UOcBwUaaUZOEN19uGGOdZKSTO48coDk0s1CacHLK3JfTX7YIkQA3zxAx0l-ImQMs7hex-_2rhPXDPzDkdhueN1VNq-iqWvhQp0d1oukKu5NwIsB2-cGDRjmd-FZzBYubT60OJFkuQnGR3-NNdGSxMUkM9j_-vQsXLR_lkemqlRld-ATnUsrCtV5ejw5HIrrB7DvqXh54L2t8wx1WLvYq29-LMguqVaAY34jqMIuMHOhT_lAADCC0ydevrRDBQtnmKfbmStOnOfXjfII5C12srYKo7NH1qYz6BmNdFe7ZIt4glUvm9SpnggeE-vrWpUxj3-b_PRfB63AcXApvphzxbz_Q-O_v0sXji_TWgasj_mK0)


## 2. Creation of a new `theme` or a new `skin`

Now the big deal will.... with which name:
- What name would we give to this `theme` or `skin`?

Hints:
- `old-plantuml`
- `legacy-plantuml`
- `fefece` _(in memory of the characteristic color `#fefece`! 😄)_

The poll is open...

Regards.

**Comment by arnaudroques** [comment, marked as answer]
There is already [a more simple option](http://www.plantuml.com/plantuml/umla/SoWkIImgAStDuIhEpinJACelJkLAJ2x9Br9mpCbCJkK2amgwTWfE-KbM2XwfEJdvN0wfUIb07G00):

```
@startuml
skin rose
actor Alice
Alice -> Bob: Hello
@enduml
```

Not yet documented...

**Reply by The-Lum** [reply]
Hello all,

Very good solution...
_I really need to ask for a plantUML training...
PlantUML is really full of Easter eggs... 🎉_ 

Perhaps, in complement of `rose`, add also another name for `rose`, for better understanding... _`old-plantuml`, `legacy-plantuml`, `fefece`, ..._

Regards.

**Reply by hanlins** [reply]
Cool! At least I can get my favorite skin back 😄
> Not yet documented...

😢
